### ISSUE TITLE
Please add Documentation for using Custom Objects

### ISSUE THREAD

**Issue by rakeshmothukuru1** [tensorflow/serving#1683]
Custom Objects is a very good option for the Tensorflow Developers. It would be great if a Tutorial is present in which a Model is built using Custom Layers, uses Custom Metrics, Custom Loss, Custom Optimizers, etc.. and **then performs inference using Tensorflow Serving**.

It will help as a reference guide for all the Developers using Custom Objects in their Projects. Thanks!

**Comment by shadowdragon89** [comment]
misterpeddy@, do you know any one who is familiar with the context here?

**Comment by misterpeddy** [comment]
I'm not sure what the question means. Any subclass of tf.keras.Model can be serialized to SavedModel and served by TensorFlow Serving. I suggest closing this unless original user posts any clarifications.

**Comment by rakeshmothukuru1** [comment]
@misterpeddy,
I agree to close this issue unless there is difference in the process of `Inference` and `Building the Client File` while using the `Custom Objects`. Thanks!

**Comment by misterpeddy** [comment]
It doesn't make sense to me that there'd be a difference but if you have an example where there is a difference, please do post it.
